cmake/custom/tests.cmake

```diff
@@ -1,6 +1,13 @@
+#.rst:
+#
+# autocmake.yml configuration::
+#
+#   docopt: "--enable-tests Enable tests [default: False]."
+#   define: "'-DENABLE_TESTS={0}'.format(arguments['--enable-tests'])"
+
 option(ENABLE_TESTS "Enable test suite" ON)
```

Comment:
Consider using the `option_with_print` function from Autocmake (see previous comment)